src/peft/tuners/ia3/bnb.py

```diff
@@ -70,3 +70,59 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
             result = result.to(expected_dtype)
 
         return result
+
+
+class Linear4bit(bnb.nn.Linear4bit, IA3Layer):
```

Comment:
I noticed that I made a mistake while refactoring because I didn't put a guard on the definition of the IA³ bnb layers. Would you be so kind as to add them? See the `lora/bnb.py` module for reference.